keyboards/handwired/ortho60/ortho60.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_1x2uC( \
+    k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, \
+    k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, \
+    k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, \
+    k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, \
+    k40, k41, k42, k43, k44, k45, k46, k47, k48, k49, k4a, k4b \
```

Comment:
`k46` isn't defined in your matrix below, so it needs to be removed from the layout here. Otherwise, attempts to compile using this macro will fail with errors.

```suggestion
    k40, k41, k42, k43, k44,    k45,   k47, k48, k49, k4a, k4b \
```

Reply:
Thank you -this has been fixed